src/views/Login.vue

```diff
@@ -0,0 +1,86 @@
+<template>
```

Comment:
use pug for this template

Reply:
Will do it in a specific PR or I will have problems with the other open PRs.